app/services/action_limiter.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class ActionLimiter
+  class ActionLimitsExceededError < StandardError
+    def initialize(*)
+      super("Rate limit exceeded")
+    end
+  end
+
+  TIME_SPAN = 1.hour
+  BUCKET_INTERVAL = 10.minutes
+  THRESHOLD = 10
```

Comment:
Should this be from a config file instead? @duduribeiro @guicassolato @hallelujah

Reply:
Yes we can and also we need sane constants defaults in case the config does not exists ;)

Reply:
imo if the config does not exist, the feature is disabled :smile: